### ISSUE TITLE
environment.yml should list pip

### ISSUE THREAD

**Issue by KrisThielemans** [SyneRBI/SIRF-Exercises#202]
I get a warning
```
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
```

**Comment by KrisThielemans** [comment]
sorry, `environment.yml` is actually only introduced in #200

**Comment by KrisThielemans** [comment]
This is fixed in #200
